hoomd/Analyzer.h

```diff
@@ -130,7 +143,8 @@ class PYBIND11_EXPORT Analyzer
     std::shared_ptr<const ExecutionConfiguration>
         m_exec_conf; //!< Stored shared ptr to the execution configuration
     std::vector<std::shared_ptr<hoomd::detail::SignalSlot>>
-        m_slots; //!< Stored shared ptr to the system signals
+        m_slots;                        //!< Stored shared ptr to the system signals
+    std::shared_ptr<Trigger> m_trigger; /// Trigger that determines if updater runs.
```

Comment:
```suggestion
    /// Trigger that determines if updater runs.
    std::shared_ptr<Trigger> m_trigger; 
```

The `///` comment syntax documents the following line.